.docker/paste/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM rustlang/rust:nightly
+
+MAINTAINER Jos Ahrens <zarthus@liefland.net>
+WORKDIR /app
+
+RUN apt update -y
+RUN apt install -y libsodium-dev cmake build-essential
+
+ADD bashrc.sh /root/.bashrc
+# TODO: get it from .env instead? but then it's not accessable by the container.
+ADD postgres_env /root/postgres_env
+RUN sed -E 's/[^=]+=//g' /root/postgres_env | tr "\n" " " | awk '{ print "db:5432:" $1 ":" $2 ":" $3 }' > /root/.pgpass && chmod 0600 /root/.pgpass
```

Comment:
that is relying on the parameters being in the right order, since they are named, it's counter intuitive.